solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -653,6 +661,28 @@
                 "[variables('eventHubResourceId')]"
             ]
         },
+        {
+            "name": "[parameters('provisioningServiceName')]",
+            "type": "Microsoft.Devices/provisioningServices",
+            "apiVersion": "[variables('provisioningServiceApiVersion')]",
+            "location": "[variables('location')]",
+            "tags": {
+                "IotSuiteType": "[parameters('solutionType')]"
+            },
+            "properties": {
+                "iotHubs": [
+                    {
+                        "connectionString": "[concat('HostName=', reference(variables('iotHubResourceId')).hostName, ';SharedAccessKeyName=', variables('iotHubKeyName'), ';SharedAccessKey=', listkeys(variables('iotHubKeyResource'), variables('iotHubApiVersion')).primaryKey)]",
+                        "location": "[variables('location')]"
+                    }
+                ]
+            },
+            "sku": {
+                "name": "S1",
+                "capacity": 1
```

Comment:
Does this capacity needs to match the IotHub Skus? What if the parameter for sku is S2 or S3?

Reply:
This is different than capacity parameter of IotHub Skus and represents the number of DPS units to provision.